gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-all.zip
```

Comment:
There is a second gradle-wrapper.properties at https://github.com/opensearch-project/security/blob/main/bwc-test/gradle/wrapper/gradle-wrapper.properties